examples/plot_simulate_lfp_array.py

```diff
@@ -0,0 +1,106 @@
+# params_fname = op.join(hnn_core_root, 'param', 'default.json')
+params_fname = op.join(hnn_core_root, 'param', 'gamma_L5weak_L2weak.json')
+params = read_params(params_fname)
+params['tstop'] = 250
+###############################################################################
+# We will start with simulating the evoked response from
+# :ref:`evoked example <sphx_glr_auto_examples_plot_simulate_evoked.py>`.
+# We first instantiate the network.
+# (Note: Setting ``add_drives_from_params=True`` loads a set of predefined
+# drives without the drives API shown previously).
+net = Network(params)
+weights_ampa = {'L2_pyramidal': 0.0008, 'L5_pyramidal': 0.0075}
+synaptic_delays = {'L2_pyramidal': 0.1, 'L5_pyramidal': 1.0}
+rate_constant = {'L2_pyramidal': 140.0, 'L5_pyramidal': 40.0}
+net.add_poisson_drive(
+    'poisson', rate_constant=rate_constant, weights_ampa=weights_ampa,
+    location='proximal', synaptic_delays=synaptic_delays, seedcore=1079)
+
+###############################################################################
+# LFP recordings require specifying the electrode postion. It can be useful
+# to visualize the cells of the network to decide on the placement of each
+# electrode.
+net.plot_cells()
+
+###############################################################################
+# Electrode positions are stored under ``Network.lfp_array`` as a list
+# of tuples. Once we have chosen x,y,z coordinates for each electrode, we can
+# add them to the simulation.
+
+# XXX coordinates FUBAR, cortical depth direction is Y in NEURON objects!
+# electrode_pos = [(2, 400, 2), (6, 800, 6)]
+depths = list(range(-400, 1900, 200))
+electrode_pos = [(45, dep, 45) for dep in depths]
+net.add_electrode_array('arr1', electrode_pos)
+print(net.lfp_array)
+net.plot_cells()
+
+# with MPIBackend(n_procs=4):
+dpl = simulate_dipole(net)
+###############################################################################
+# Now that our electrodes are specified, we can run the simulation. The LFP
+# recordings are stored under ``Network.lfp_array``.
+import matplotlib.pyplot as plt
+
+plt.figure()
+trial_idx = 0
+plt.plot(net.lfp_array['arr1'].times,
+         net.lfp_array['arr1'].voltages[trial_idx][1])
+plt.plot(net.lfp_array['arr1'].times,
+         net.lfp_array['arr1'].voltages[trial_idx][-2])
+plt.legend([f'e_pos {electrode_pos[1]}', f'e_pos {electrode_pos[-2]}'])
+plt.xlabel('Time (ms)')
+plt.ylabel(r'Potential ($\mu V$)')
+plt.show()
+
+###############################################################################
+# We can compare the dipole current wave form to that of a single LFP channel.
+show_electrode = [6, 10]
+tmin, tmax = 50, 225
+window_len = 10  # ms
+decimate = [5, 4]  # from 40k to 8k to 2k
+fig, ax = plt.subplots(1, 1, sharex=True, figsize=(8, 6))
+lfp_ax = ax.twinx()
+
+dpl[trial_idx].copy().smooth(
+    window_len=window_len).plot(ax=ax, show=False, decim=decimate,
+                                tmin=tmin, tmax=tmax, color='k')
```

Comment:
this waveform looks very different from the gamma example?!

Reply:
It should be identical. Note that the gamma example doesn't smooth, whereas here there's a 10 ms Hamming window applied. Does this explain the apparent difference?